### PR TITLE
Feature/add postman collection

### PR DESCRIPTION
**Pull request by ElRaptorus**
## What did you change?

Add basic postman collection and environment, covering each request that the process-engine-server can currently handle.

## How can others test the changes?

Import the Collection and Environment into postman, start a process-engine-server and run the requests.

Important:
By default, route variables (like :id) are only set with placeholders!

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: Process Engine.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+					"path": [
+						"messagebus",
+						"publish"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "/processengine/start",
```
Comment (5ebastianMeier) [inline Process Engine.postman_collection.json]: The body for this post message should contain an example - e.g. it contains a `msg` property of type `IParamsStart` (see process_engine_http/process_engine_controller and process_engine/process_engine_service for details)

**Comment by ElRaptorus** [comment]
@5ebastianMeier done

**Comment by cmg-dev** [comment]
@ElRaptorus Was ist denn hiermit?

**Comment by ElRaptorus** [comment]
@cmg-dev Da hatte ich noch auf approval von @ppoetschke gewartet.

File: Process Engine.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+						"value": "application/json"
+					},
+					{
+						"key": "Authorization",
+						"value": "Bearer {{bearer_token}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n\t\"username\": \"admin\",\n\t\"password\": \"admin\"\n}"
```
Comment (ppoetschke) [inline Process Engine.postman_collection.json]: Hier sind einige Default-Werte für den Body der Requests drin, die keinen Sinn machen bzw. nicht zum Request passen. Hauptsächlich `username, password` props.

**Comment by ElRaptorus** [comment]
@ppoetschke done

**Comment by cmg-dev** [comment]
@ppoetschke Please have a look again, then we can merge this :)

**Pull request merged**